### PR TITLE
Test: add minimum coverage based on current main branch coverage

### PR DESCRIPTION
**Pull request by jdkandersson**
Small change to get in compliance with new coding standard

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,6 +10,7 @@ branch = true
 
 [tool.coverage.report]
 show_missing = true
+fail_under = 100
```
Comment (merkata) [inline pyproject.toml:13]: It's a high standard :smile: 
I think it's excellent to achieve 100, but could fail if under 85%. It is what we discussed with @mthaddon and I'm more in favor of that. I'm totally fine with this as well, I think however we decide to do it, it should be the same for all charms we develop and maintain.

Comment (jdkandersson) [reply pyproject.toml:13]: The coverage is already 100% on the main branch, in line with the standard (which is the maximum of 85% and the coverage on the main branch): https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-coverage

Comment (jdkandersson) [reply pyproject.toml:13]: Indico is sitting at 90%: https://github.com/canonical/indico-operator/blob/main/pyproject.toml

Comment (merkata) [reply pyproject.toml:13]: > The coverage is already 100% on the main branch, in line with the standard (which is the maximum of 85% and the coverage on the main branch): https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-coverage

Sorry, you're right. I just went through the guide and it has the maximum of 85% and the percentage in the main branch, so this is well aligned. Thanks for taking the time to clarify.

**Review by merkata — APPROVED** [review]
Change is good and aligned with our coding standards.

**Pull request merged**